deploy/docker/fs/opt/appsmith/utils/bin/import_db.js

```diff
@@ -1,47 +1,50 @@
-// Init function export mongodb
-const shell = require('shelljs');
 const readlineSync = require('readline-sync');
 const process = require('process');
 const Constants = require('./constants');
 const utils = require('./utils');
 
-
-function import_database() {
-  console.log('import_database  ....')
-  dbUrl = utils.getDburl();
-  const cmd = `mongorestore --uri='${dbUrl}' --drop --archive='${Constants.RESTORE_PATH}/${Constants.DUMP_FILE_NAME}' --gzip`
-  shell.exec(cmd)
-  console.log('import_database done')
-}
-
-function stop_application() {
-  shell.exec('/usr/bin/supervisorctl stop backend rts')
-}
-
-function start_application() {
-  shell.exec('/usr/bin/supervisorctl start backend rts')
+async function importDatabase() {
+  console.log("import_database  ....");
```

Comment:
Can we improve the console logs here? I'm only suggesting this because users will read these logs as well. 
```suggestion
  console.log("Importing the database");
```

Reply:
I was going to go over all logs in a future PR, but sure. 🙂